### ISSUE TITLE
day_1/2_classify_names_with_MLP & test set evaluation

### ISSUE THREAD

**Issue by braingineer** [joosthub/pytorch-nlp-tutorial-ny2018#3]
Test set section is currently looping over `dataloader` and it should be `dataloader_test`
